ui/templates/grade_record.html

```diff
@@ -34,6 +34,8 @@
         </div>
 
         <div class="school-name">
+          <img src="{% static 'images/edx_logo.png' %}" alt="edX" class="edx-logo">
+          <span class="pipe">|</span>
           <img src="{% static 'images/mitx_logo.png' %}" alt="MITx Logo" />
```

Comment:
I got a request for it to be "MITx on edX", like this: 

```suggestion
         <img src="{% static 'images/mitx_logo.png' %}" alt="MITx Logo" />
         <span class="pipe">on</span>
         <img src="{% static 'images/edx_logo.png' %}" alt="edX" class="edx-logo">
 ```